packages/boot/test/fixtures/application.ts

```diff
@@ -5,13 +5,14 @@
 
 import {RestApplication} from '@loopback/rest';
 import {ApplicationConfig} from '@loopback/core';
+// tslint:disable-next-line:no-unused-variable
+import {RepositoryMixin, Class, Repository} from '@loopback/repository';
 
-/* tslint:disable:no-unused-variable */
 // Binding and Booter imports are required to infer types for BootMixin!
+// tslint:disable-next-line:no-unused-variable
```

Comment:
Ditto here. Are you saying this is needed to make `npm run lint` pass? Could you please double-check?